webdriver/tests/bidi/script/get_realms/context.py

```diff
@@ -29,17 +27,14 @@ async def test_context(
 
     result = await bidi_session.script.get_realms(context=new_context["context"])
 
-    recursive_compare(
-        [
-            {
-                "context": new_context["context"],
-                "origin": test_origin,
-                "realm": new_context_result["realm"],
-                "type": "window",
-            },
-        ],
-        result,
-    )
+    assert [
+        {
+            "context": new_context["context"],
+            "origin": test_origin,
+            "realm": new_context_result["realm"],
+            "type": "window",
+        },
+    ] <= result
```

Comment:
Perhaps this should solve the issue, but I'm not sure if it works. Another option is to keep arrays assertions strict:
```suggestion
    assert [
        AnyLE({
            "context": new_context["context"],
            "origin": test_origin,
            "realm": new_context_result["realm"],
            "type": "window",
        }),
    ] == result
```

Reply:
Done